dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -2867,14 +2868,23 @@ BlockInputStreamPtr Segment::getBitmapFilterInputStream(const DMContext & dm_con
     {
         return std::make_shared<EmptyBlockInputStream>(toEmptyBlock(columns_to_read));
     }
+
+    // buildBitmapFilter will only read handle, version and tag.
+    const ColumnDefines build_bitmap_columns{
+        getExtraHandleColumnDefine(is_common_handle),
+        getVersionColumnDefine(),
+        getTagColumnDefine(),
+    };
+    auto build_bitmap_block_rows = clipBlockRows(dm_context.db_context, expected_block_size, build_bitmap_columns);
```

Comment:
There is no need to clip, just pass `expected_block_size` is ok.

Reply:
The memory usage of get bitmap is small, so I think pass `expected_block_size` is acceptable.

Reply:
And those data will not pass to compute layer.